webrender/src/prim_store.rs

```diff
@@ -1620,29 +1645,37 @@ impl PrimitiveInstance {
             clipped_world_rect: None,
             #[cfg(debug_assertions)]
             prepared_frame_id: FrameId(0),
+            #[cfg(debug_assertions)]
+            id: PrimitiveId(NEXT_PRIM_ID.fetch_add(1, Ordering::Relaxed)),
             clip_task_id: None,
             gpu_location: GpuCacheHandle::new(),
             opacity: PrimitiveOpacity::translucent(),
             clip_chain_id,
             spatial_node_index,
         }
     }
+
+    #[cfg(debug_assertions)]
+    pub fn is_chased(&self) -> bool {
+        PRIM_CHASE_ID.load(Ordering::SeqCst) == self.id.0
+    }
+
+    #[cfg(not(debug_assertions))]
+    pub fn is_chased(&self) -> bool {
```

Comment:
May be worth to `#[inline]` so that we ensure the compiler can remove the dead code.